src/OrbitQt/orbitmainwindow.ui

```diff
@@ -58,7 +58,7 @@ QPushButton:disabled {
         <string>MainTabWidget</string>
        </property>
        <property name="currentIndex">
-        <number>0</number>
+        <number>1</number>
```

Comment:
Is this intended and does it make a difference after startup?

Reply:
Thanks for pointing out this!!!! This is not intended. I think when I editing the ui file in qtdesigner, I clicked the capture window but forgot to change back to the home window.